config/staging-config.json

```diff
@@ -112,4 +112,4 @@
         "firstName": "",
         "lastName": ""
     }
-}
+}
```

Comment:
looks as if config/staging-config.json was already present in zaynab's changes? I don't think i saw that but here I just see 1 change as a single line delete.

Reply:
That new line removal is because of my IDE setup. Fixed here: [484e7bf](https://github.com/orica-it-enterprise/cortex-data-foundation/pull/2/commits/484e7bfd581603495607cefbe403dca734ae55c5)

The diff will now just show a rename

Reply:
okay, thanks